resources/views/welcome.blade.php

```diff
@@ -1,10 +1,487 @@
 @extends('layouts.master')
 
 @section('main')
-<div class="container mx-auto">
-    <div class="font-bold text-lg">
-        <a class="px-4 py-2 text-white bg-red-500 rounded shadow-inner mr-2" href="/google/login">Login with Google</a>
-        <a class="px-4 py-2 text-white bg-blue-500 rounded shadow-inner" href="/facebook/login">Login with Facebook</a>
+
+<div class="sm:text-xs block ">
+
+    <div class="lg:w-1/2 md:w-3/4 w-4/5 lg:absolute-center-40 absolute-center" style="font-family: 'Major Mono Display', monospace;">
+      <svg version="1.1" viewBox="0 0 600 600" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <text x="50%" y="50%" text-anchor="middle" dominant-baseline="middle" font-size="5rem" fill="#ffdc34" >CHAKRAVYUH</text>
+      </svg>
     </div>
+    <div class="lg:w-1/3 md:w-1/2 w-3/5 mx-auto lg:fixed-center-40 fixed-center">
+      <svg class="md:opacity-25 opacity-50" viewBox="0 0 681 681" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path id="path1" d="M639.005 340.5C639.005 295.159 628.711 250.41 608.898 209.627C589.086 168.845 560.272 133.092 524.63 105.067C488.989 77.0408 447.449 57.4732 403.144 47.8391C358.838 38.2051 315.561 38.3055 271.5 49L276.869 73.2419C241.772 81.4674 208.639 96.5252 179.362 117.556C150.084 138.586 125.236 165.177 106.235 195.811C87.2338 226.444 74.4527 260.52 68.6211 296.093C62.7895 331.666 64.0217 368.04 72.2472 403.137L96.4604 396.886C86.8858 355.617 87.9779 312.588 99.6336 271.857C111.289 231.126 133.125 194.034 163.082 164.077L180.76 181.755C206.834 155.68 238.93 136.43 274.21 125.705C309.49 114.98 346.87 113.111 383.044 120.262C419.218 127.414 453.074 143.366 481.62 166.708C510.166 190.051 533.809 223.466 548 257.5L519.964 267.941C531.674 297.063 536.161 328.586 533.044 359.818L493.121 357.012C490.306 383.196 480.845 408.227 465.636 429.727C450.426 451.226 429.974 468.482 406.22 479.854C382.466 491.226 356.2 496.337 329.915 494.702C303.63 493.067 278.2 484.741 256.038 470.513L269.026 443.13C285.936 454.743 305.454 461.997 325.843 464.248L336.72 419.951C319.829 419.359 303.567 413.4 290.293 402.937C277.019 392.475 267.425 378.054 262.903 361.769C258.381 345.483 259.168 328.181 265.148 312.372C271.127 296.564 281.99 283.073 296.158 273.858L311.114 299.952C316.439 296.224 322.446 293.58 328.792 292.174C335.138 290.767 341.7 290.623 348.101 291.752C354.503 292.881 360.62 295.26 366.102 298.752C371.584 302.245 376.325 306.783 380.054 312.108" stroke="white" stroke-opacity="0.5"/>
```

Comment:
try to do this in an efficient way, don't repeat yourself. You can use `partials` in laravel and include it in this blade file, take @shubham-pal help on that.